nvflare/private/fed/client/client_runner.py

```diff
@@ -278,8 +300,17 @@ def _process_task(self, task: TaskAssignment, fl_ctx: FLContext) -> Shareable:
                 fl_ctx=fl_ctx,
                 msg=f"submit result: {ReturnCode.EXECUTION_RESULT_ERROR}",
             )
+        except UnsafeJobError:
+            self.log_exception(fl_ctx, f"UnsafeJobError from executor {executor_name}")
+            executor.unsafe = True
```

Comment:
I think it might be good to have the reason print out:

```
except UnsafeJobError as e:
    self.log_exception(fl_ctx, f"UnsafeJobError ({secure_format_exception(e)}) from executor {executor_name}")
```

Reply:
Done.